src/components/Details/VmDetails/VmDetails.js

```diff
@@ -50,14 +50,20 @@ export class VmDetails extends React.Component {
       k8sError: null,
       templateError: null,
       form: {},
+      preEditFormValues: {},
```

Comment:
nit: can we call it `preEditForm`? So it has the same shape as `form` - it is less confusing IMO.

Reply:
Done. I struggled with what to call it. I like yours better. =)